server/badge.go

```diff
@@ -42,7 +43,8 @@ func GetBadge(c *gin.Context) {
 
 	build, err := store.GetBuildLast(c, repo, branch)
 	if err != nil {
-		c.String(404, badgeNone)
+		log.Warning(err)
```

Comment:
actually I'll just adjust this later. I need to go through an audit logging anyway and I don't want to delay the PR :)